DataPluginDemo.cs

```diff
@@ -6916,6 +6924,8 @@ public void Init(PluginManager pluginManager)
             pluginManager.AddProperty("DDUEnabled", this.GetType(), Settings.DDUEnabled);
             pluginManager.AddProperty("SW1Enabled", this.GetType(), Settings.SW1Enabled);
             pluginManager.AddProperty("DashLEDEnabled", this.GetType(), Settings.DashLEDEnabled);
+            pluginManager.AddProperty("ShowMap", this.GetType(), Settings.ShowMap);
```

Comment:
This seems like a boolean - maybe `ShowMapEnabled`

Reply:
I will rename it accordingly